unfoldingWord/translationCore
# added tokenizer in stringHelpers and tests

#### This pull request addresses:

Added a tokenizer in stringHelpers, so we can consistently turn a string into an array of words.


#### How to test this pull request:

- Run tests to make sure they pass, in particular, stringHelpers.
- Review tests to cover use cases.
- Run alignment tool on a compatible project, make sure word bank doesn't have punctuation in it.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2810)
